FeedReader/FeedReader.csproj

```diff
@@ -1,31 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net452</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net452;netcoreapp2.1</TargetFrameworks>
```

Comment:
Not sure if yall want this bumped to 2.0 or not. Let me know and Ill move it back down(it should be backwards compatible so it should matter). I just needed core 2.1